wee_slack.py

```diff
@@ -2113,6 +2113,14 @@ def __init__(self, message_json, team, channel, override_sender=None):
                 message_json['text']
             )
 
+        if message_json.get('upload'):
+            parts = [message_json['text']]
+            for f in message_json.get('files', []):
+                parts.append("({}) - {}".format(
+                    f.get('title'),
+                    f.get('url_private')))
+            message_json['text'] = ' | '.join(p for p in parts if p)
```

Comment:
We should append this in the `render` method, like we do with `unwrap_attachments`, instead of changing the message text.

We also need to handle message changes, so the file doesn't disappear when the message is edited. This can also be done the same way as for `unwrap_attachments`.